### PR TITLE
last references to run-protocol

### PR DESCRIPTION
**Pull request by turadg**
Missed a few in #5803

### CODE REVIEW

File: packages/cosmic-swingset/Makefile [hunk 1]
```diff
@@ -71,8 +71,8 @@ gov-q:
 	$(AGCH) query gov proposals --output json | \
 		jq -c '.proposals[] | [.proposal_id,.voting_end_time,.status]'
 
-deploy-runStake: ../run-protocol/test/runStake/runStake-deploy.js
-	echo agoric deploy ../run-protocol/test/runStake/runStake-deploy.js
+deploy-runStake: ../inter-protocol/test/runStake/runStake-deploy.js
+	echo agoric deploy ../inter-protocol/test/runStake/runStake-deploy.js
```
Comment (dckc) [inline packages/cosmic-swingset/Makefile]: Please delete this obsolete Makefile rule.

There is no `runStake-deploy.js` file in the repo. I suspect I had something by that name locally at the time, but I don't even have that. I suppose it's obsolete in favor of stuff like https://github.com/Agoric/agoric-sdk/blob/master/packages/inter-protocol/src/proposals/core-proposal.js

**Review by dckc — APPROVED** [review]
runStake-deploy.js is obsolete; I prefer that we remove the reference to it.

Not critical, though.

File: packages/cosmic-swingset/Makefile (hunk 1 above)
Comment (turadg) [reply packages/cosmic-swingset/Makefile]: Oh good. `measure.js` may be similarly dead so I'll wait for an answer on that.

**Review by Chris-Hibbert — APPROVED** [review]
I don't have an opinion on measure.js, but if it stays, this change is necessary.

**Comment by warner** [comment]
`measure.js` (and indeed most of `packages/SwingSet/misc-tools/`) is an ad-hoc tool we put together a while ago to get a sense of how big a computron is; how much meter is consumed by various operations. For some reason it reminds me of the little "how to think in metric" booklet I got as a child, with pictures of different items and how much they weighed/etc in an unfamiliar set of units.

I'd like to retain those tools, even if they're somewhat bit-rotted, as it will make it that much easier for us to dust off the next time we want to ask those questions. I'm ok if they don't run right now, but e.g. updating the paths to match changes elsewhere in the SDK will keep them fresh-*er*, and is exactly why I keep them checked in.

**Review by warner — APPROVED** [review]
`measure.js` change looks good to me, thanks

**Comment by turadg** [comment]
This PR [failed integration test](https://github.com/Agoric/agoric-sdk/runs/7506282693?check_suite_focus=true) but I don't see why.

**Pull request merged**